src/bundle/Resources/public/scss/_mixins.scss

```diff
@@ -0,0 +1,68 @@
+@mixin amazing-checkbox($size: 2rem) {
```

Comment:
I think the `amazing-checkbox` is the coolest name ever but we cannot use it. We will implement another checkbox with icons.

Reply:
Ok, I'll use the `checkbox-switcher` as the mixin name.